templates/data/postgresql.html

```diff
@@ -1,481 +1,561 @@
 {% extends 'base_index.html' %}
 
-{% block meta_copydoc %}https://docs.google.com/document/d/1GxineF4WLt0YlShaYkrzqWjIOkXjLJEfsnUdWVfWnz4/edit{% endblock meta_copydoc %}
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1GxineF4WLt0YlShaYkrzqWjIOkXjLJEfsnUdWVfWnz4/edit
+{% endblock meta_copydoc %}
 
-{% block title %}Enterprise-grade solutions for PostgreSQL databases{% endblock %}
+{% block title %}An advanced, enterprise-grade PostgreSQL solution from Canonical{% endblock %}
 
-{% block meta_description %}Get secure and automated deployment, maintenance and upgrades for your PostgreSQL databases across private and public clouds.{% endblock %}
+{% block meta_description %}
+  Charmed PostgreSQL comes with up to 10 years of support and security maintenance for PostgreSQL. It helps you secure and automate the deployment, maintenance and upgrades of your PostgreSQL databases across private and public clouds.
+{% endblock %}
 
-{% block body_class %}is-paper{% endblock body_class %}
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
+  <section class="p-section--hero">
+    <div class="row">
+      <div class="col-3 col-medium-2">
+        <div class="u-hide--small" style="margin-top: -75px">
+          {{ image(url="https://assets.ubuntu.com/v1/176b241d-postgresql-logo-mark.png",
```

Comment:
We have a `p-image-wrapper` custom class on u.com for logo images like these. Can you wrap this image in something similar for c.com?

Reply:
Wrapping it in a image class will make it centered which will not align with design.

Reply:
Can you see how we are arranging other hero logos on c.com? When hovered over the logo the container is too large, I don't think this should be the desired behaviour.
<img width="1004" alt="Screenshot 2024-09-06 at 11 53 53 AM" src="https://github.com/user-attachments/assets/6cde4ca7-e5f1-44ba-a222-5153c56c939e">

Reply:
The image itself is large here and this has been done to put it on top-right corner. A negative margin is being used here.
Similar to here in opensearch
![image](https://github.com/user-attachments/assets/5f65ac3b-f1de-49b4-b859-e591d3bdcaa2)